src/Laravel/DiscoveryController.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace OpenIDConnect\Laravel;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Route;
+
+class DiscoveryController
+{
+    public function discovery(Request $request)
```

Comment:
Can't we use an invocable controller while calling a single method?
```suggestion
    public function __invoke(Request $request)
```

Reply:
Done!